internal/dockerutil/setup.go

```diff
@@ -92,6 +92,7 @@ func DockerSetup(t DockerSetupTestingT) (*client.Client, string) {
 func dockerCleanup(t DockerSetupTestingT, cli *client.Client) func() {
 	return func() {
 		showContainerLogs := os.Getenv("SHOW_CONTAINER_LOGS") != ""
+		failedLogging := os.Getenv("DISABLE_FAILED_LOGGING") == ""
 		containerLogTail := os.Getenv("CONTAINER_LOG_TAIL")
```

Comment:
Ok, this is me being really nit picky I'm sorry! But these two ENV's do really similar things, but code readability it's really confusing. 

I also don't think this fix is necessary anymore. But if we want this feature it how about...

Maybe instead of just checking if `SHOW_CONTAINER_LOGS` is "", we check:
- "" -> default which only shows logs on failure
- "always" -> always shows logs, both for success and failure
- "never" -> never shows logs

Reply:
I like this approach much more, will do this & add to docs

Reply:
Completed in fe3d5d06e15780bfddc2660d9890259ce7087a0c